controller/src/view/world.rs

```diff
@@ -182,4 +183,58 @@ impl ViewController {
             }
         }
     }
+
+    pub fn draw_health_bar(
+        &self,
+        draw: &imgui::DrawListMut,
+        bar_x: f32,
+        bar_y: f32,
+        filled_height: f32,
+        bar_width: f32,
+        health_color: [f32; 4],
+        border_thickness: f32,
+        border_color: [f32; 4],
+        bar_height: f32,
+        border_bar_y: f32,
+    ) {
+        draw.add_rect_filled_multicolor(
```

Comment:
Why not:
```
draw.add_rect([vmin.x, vmin.y], [vmax.x, vmax.y], *esp_color)
                                .filled(true)
                                .build();
```
instead of `add_rect_filled_multicolor`?